apps/portal/src/pages/_app.tsx

```diff
@@ -11,8 +11,8 @@ import AppShell from '~/components/global/AppShell';
 
 import type { AppRouter } from '~/server/router';
 
-import '~/styles/globals.css';
```

Comment:
Did this ordering cause any bug?

Reply:
I saw the sidebar menu disappear with this ordering, is that intended? 
<img width="1148" alt="image" src="https://user-images.githubusercontent.com/65328119/194031206-cd9d9107-f626-4769-a03b-9885985101f5.png">

Reply:
Oh wow definitely not intended. Thanks for fixing :)

Reply:
Strange thing is, the button disappears when I change the ordering and only appears onHover


https://user-images.githubusercontent.com/65328119/194038933-085d2a0c-8337-4f5d-b2ae-8915e6ac1f09.mov